docs/src/main/paradox/release-notes/index.md

```diff
@@ -1,5 +1,13 @@
 # Release Notes
 
+## 1.0.1
+A minor bug fix release. The class renaming described below (`#491`) is not expected to affect anyone
+upgrading from version 1.0.0 but it is recommended that Apache Pekko users switch to the 1.0.1 release when it becomes available.
```

Comment:
Can we make the language a bit stronger when it comes to encouraging people to upgrade?

Also capitalize the upgrading.

Reply:
since the text is not agreed - I will revert to using cwiki for the draft release notes - it is messy to being causing big CI builds while tweaking the text

https://cwiki.apache.org/confluence/display/PEKKO/Pekko+Core+Release+Notes